quasar/src/components/editor/QEditor.json

```diff
@@ -0,0 +1,226 @@
+        },
+        "key": {
+          "type": "Number",
+          "desc": "Keycode of a key to be used as a shortcut for triggering this element"
+        },
+        "handler": {
+          "type": "Function",
+          "desc": "Either this or \"cmd\" is required. Function for when button gets clicked/tapped."
+        },
+        "cmd": {
+          "type": "String",
+          "desc": "Either this or \"handler\" is required. This must be a valid execCommand method according to the designMode API."
+        },
+        "param": {
+          "type": ["String", "Function"],
+          "desc": "Only set a param if using a \"cmd\". This is commonly text or HTML to inject, but is highly dependent upon the specific cmd being called."
+        },
+        "disable": {
+          "type": ["Boolean", "Function"],
+          "desc": "Is button disabled? If specifying a function, return a Boolean value."
+        },
+        "type": {
+          "type": "String",
+          "desc": "Pass the value \"no-state\" if the button should not have an \"active\" state",
+          "values": [null, "no-state"]
+        }
+      }
+    },
+    "fonts": {
+      "type": "Object",
+      "desc": "Object with definitions of fonts",
+      "examples": [
+        ":fonts=\"{",
+        "  arial: 'Arial',",
+        "  arial_black: 'Arial Black',",
+        "  comic_sans: 'Comic Sans MS'",
+        "}\""
+      ]
+    },
+    "toolbar": {
+      "type": "Array",
+      "desc": "An array of arrays of Objects/Strings that you use to define the construction of the elements and commands available in the toolbar",
+      "default": [
+        ["left", "center", "right", "justify"],
+        ["bold", "italic", "underline", "strike"],
+        ["undo", "redo"]
+      ]
+    },
+    "toolbar-color": {
+      "type": "String",
+      "desc": "Font color (from the Quasar Palette) of buttons and text in the toolbar"
+    },
+    "toolbar-text-color": {
+      "type": "String",
+      "desc": "Text color (from the Quasar Palette) of toolbar commands"
+    },
+    "toolbar-toggle-color": {
+      "type": "String",
+      "desc": "Choose the active color (from the Quasar Palette) of toolbar commands button",
+      "default": "primary"
+    },
+    "toolbar-bg": {
+      "type": "String",
+      "desc": "Toolbar background color (from Quasar Palette)",
+      "default": "grey-3"
+    },
+    "toolbar-flat": {
+      "type": "Boolean",
+      "desc": "Toolbar buttons are rendered \"flat\""
+    },
+    "toolbar-outline": {
+      "type": "Boolean",
+      "desc": "Toolbar buttons are rendered \"outlined\""
+    },
+    "toolbar-push": {
+      "type": "Boolean",
+      "desc": "Toolbar buttons are rendered as a \"push-button\" type"
+    },
+    "toolbar-rounded": {
+      "type": "Boolean",
+      "desc": "Toolbar buttons are rendered \"rounded\""
+    },
+    "content-style": {
+      "type": "Object",
+      "desc": "CSS Style in Object format for the input area"
```

Comment:
Check for other "X-style" props in the other API files and pls match it here. Missing "examples" breaks build.